### ISSUE TITLE
Using ndDroplet in Different Places

### ISSUE THREAD

**Issue by mwgi2005** [Wildhoney/ngDroplet#45]
Hey, I've a problem, I'm using this API on different partials in same application, I've made a partial ndDroplet.htm and calling it in different partials via <ng-Include>, but the controller is same for handling the requests, now the problem is I'm getting this Error, I've absolutely no idea what to do with it.

Error: [$compile:multidir] Multiple directives [progressbar, progressbar (module: Main_Module)] asking for new/isolated scope on: <div class="loading-path bar" ng-class="type &amp;&amp; &quot;bar-&quot; + type" data-progressbar="" ng-model="interface.progress.percent" d="M 0,1 L 398,1 L 398,234 L 1,234 L 0,1" stroke="#D3B2D1" stroke-width="1" fill-opacity="0" style="stroke-dasharray: 392px, 392px;stroke-dashoffset: 392px;">

**Comment by Wildhoney** [comment]
That's a more of an issue with how you're writing your partials, rather than `ngDroplet`.

Please take a look at the following StackOverflow to see if it helps resolve issues: http://stackoverflow.com/questions/16148086/multiple-directives-directive1-directive2-asking-for-isolated-scope-on

If you're still having problems, please post the relevant files and I'll take a look :+1:

**Comment by mwgi2005** [comment]
Will let you know thanks

**Comment by Wildhoney** [comment]
@mwgi2005 Any luck?

**Comment by mwgi2005** [comment]
No... but I've temporarily disabled this part... but obviously this is not the solution
